jinja2/includes/icons/arrows/arrow-left.svg

```diff
@@ -1,3 +1 @@
-<svg class="icon icon-arrow-left" xmlns="http://www.w3.org/2000/svg" width="29" height="32" aria-hidden="true" viewBox="0 0 23 28" role="img">
-  <path d="M27.429 16v2.286c0 1.214-.804 2.286-2.089 2.286H12.769l5.232 5.25a2.24 2.24 0 0 1 0 3.214l-1.339 1.357c-.411.411-1 .661-1.607.661s-1.196-.25-1.625-.661L1.805 18.75c-.411-.411-.661-1-.661-1.607s.25-1.196.661-1.625L13.43 3.911c.429-.429 1.018-.679 1.625-.679s1.179.25 1.607.679l1.339 1.321c.429.429.679 1.018.679 1.625s-.25 1.196-.679 1.625l-5.232 5.232H25.34c1.286 0 2.089 1.071 2.089 2.286z"/>
-</svg>
+<svg class="icon icon-arrow-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M19 11H7.83l4.88-4.88c.39-.39.39-1.03 0-1.42-.39-.39-1.02-.39-1.41 0l-6.59 6.59c-.39.39-.39 1.02 0 1.41l6.59 6.59c.39.39 1.02.39 1.41 0 .39-.39.39-1.02 0-1.41L7.83 13H19c.55 0 1-.45 1-1s-.45-1-1-1z"/></svg>
```

Comment:
I believe `aria-hidden` and `role=img` is still relevant here. Same for the other two SVG assets.